### DISCUSSION TITLE
HTTP Digest authentication

### DISCUSSION THREAD

**Discussion by beyonlo** [miguelgrinberg/microdot#233] [General]
Hello

I see [this discussion](https://github.com/miguelgrinberg/microdot/discussions/218) about `HTTP Basic authentication` that are being implemented in PR #217 

Is there plans to support `HTTP Digest authentication` too?

Thank you

**Comment by miguelgrinberg** [comment]
I wasn't planning on adding Digest because it isn't as widely used, but it wouldn't be hard to do it, even if you do it as part of your application. My [Flask-HTTPAuth](https://github.com/miguelgrinberg/flask-httpauth) package has all the logic for it, if you are interested.
